server/api/users/users.controller.ts

```diff
@@ -38,3 +38,24 @@ export const getUserById: RequestHandler<{id: string}, UserProfile> = async (req
 
   res.json(user);
 };
+
+export const getUsers: RequestHandler<UserProfile[]> = async (req, res) => {
+  const users = await User.findAll({
+    attributes: [
+      'id',
+      'discord_user_id',
+      'discord_name',
+      'bio',
+      'twitter_username',
+      'linkedin_url',
+      'github_username',
+      'website',
+    ],
+  });
```

Comment:
So we do this pattern a lot it seems like - both here and down below we should just be using the lodash pick method. Can reuse the same array of allowed fields wherever we need it. Bonus - add a getter to the User model to extract these fields